templates/index.html

```diff
@@ -16,8 +16,8 @@
             })
             .then(response => response.json())
             .then(data => {
-                if (data.status === "completed") {
-                    document.getElementById("videoPlayer").src = data.video_url;
+                if (data.status === "success") {
+                    document.getElementById("videoPlayer").src = data.data.video_url;
```

Comment:
data.data typo?

Reply:
No @omeryusufyagci , in the response object that is being returned, the video_url is wrapped inside another data object. Do you have any suggestions on how to improve this ?

Reply:
It's also because data is used both in the js and the python side. You could rename the data field of the json object to `payload` which is quite std to use.